cli/src/config/templates.toml

```diff
@@ -402,3 +404,14 @@ coalesce(
   "o",
 )
 '''
+
+signed_off_by = '''
+"Signed-off-by: "
+++ if(self.author().name() == "",
+      coalesce(self.author().email(), email_placeholder),
+      self.author().name() ++ " <" ++ coalesce(self.author().email(), email_placeholder) ++ ">",
+   )
+++ "\n"
+'''
+
+gerrit_change_id = '"Change-Id: I6a6a6964" ++ self.change_id().normal_hex() ++ "\n"'
```

Comment:
nit: the current naming convention is `format_<thing>(context)` (e.g. `format_gerrit_change_id_trailer(commit)`, but I wouldn't include these aliases in this PR to avoid bikeshedding.

btw, I think sign-off should be made by user's identity (i.e. `"Signed-off-by: " ++ committer ++ "\n"`), not by the identity of the original patch author.